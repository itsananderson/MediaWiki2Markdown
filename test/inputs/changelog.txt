=== Changelog Test ===

This tests changelog handling

== Changelog ==

= 1.0 =

* Change1
* Change2

== Screenshots ==

1. Screenshots section