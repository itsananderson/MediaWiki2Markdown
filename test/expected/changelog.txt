Changelog Test
==============

This tests changelog handling

Screenshots
-----------

![Screenshots section](screenshot-1.png "Screenshots section")

*Screenshots section*

Changelog
---------

#### 1.0 ####

* Change1
* Change2